v22.2/show-backup.md

```diff
@@ -103,6 +103,21 @@ See [Show a backup with descriptor IDs](#show-a-backup-with-descriptor-ids) for
 
 The path format is `<year>/<month>/<day>-<timestamp>`.
 
+To view a list of [locality-aware backups](take-and-restore-locality-aware-backups.html), pass the endpoint [collection URI](backup.html#backup-file-urls) that is set as the `default` location with `COCKROACH_LOCALITY=default`: 
```

Comment:
This is a great addition that will help clarify using show backup with l-a backups! Thanks!

I think your content is good, I'm wondering about the locations and if we can optimize for users finding all the information they need regarding show backup + l-a backups quickly. 

One suggestion is to package your content and the example from: https://www.cockroachlabs.com/docs/v22.2/show-backup#show-a-locality-aware-backup into an include. 

On the show backup page, you could change "Show a locality-aware backup" section  to something like:

- Show locality-aware backups
- Use show backup with locality-aware backups
- View locality-aware backups

And then add your include in the section. 

You could then also add this include as a new section on the Take and Restore Locality-Aware Backups page. This page only briefly mentions show backup, and I think it could be worthwhile for user-efficiency sake given we list how to backup, restore, and take incr backups on that page.

An include will mean that any changes to this workflow in the future can be quickly replicated to both locations.

Reply:
Sounds good, creating an include to re-use code block on other pages!